sdk/communication/Azure.Communication.Email/tests/Samples/Sample1_EmailClient.cs

```diff
@@ -238,6 +238,65 @@ public void SendEmailWithAttachment()
             #endregion Snippet:Azure_Communication_Email_Send_With_Attachments
         }
 
+        [RecordedTest]
+        [SyncOnly]
+        public void SendEmailWithInlineAttachment()
+        {
+            EmailClient emailClient = CreateEmailClient();
+
+            #region Snippet:Azure_Communication_Email_Send_With_Inline_Attachments
+            // Create the email content and reference any inline attachments.
+            var emailContent = new EmailContent("This is the subject")
+            {
+                PlainText = "This is the body",
+                Html = "<html><body>This is the html body<img src=\"cid:myInlineAttachmentContentId\"></body></html>"
+            };
+
+            // Create the EmailMessage
+            var emailMessage = new EmailMessage(
+                //@@ senderAddress: "<Send email address>" // The email address of the domain registered with the Communication Services resource
+                //@@ recipientAddress: "<recipient email address>"
+                /*@@*/ senderAddress: TestEnvironment.SenderAddress,
+                /*@@*/ recipientAddress: TestEnvironment.RecipientAddress,
+                content: emailContent);
+
+#if SNIPPET
+            var filePath = "<path to your file>";
+            var attachmentName = "<name of your attachment>";
+            var contentType = MediaTypeNames.Text.Plain;
+            var contentId = "myInlineAttachmentContentId";
+#endif
+
+#if SNIPPET
+            var content = new BinaryData(System.IO.File.ReadAllBytes(filePath));
+#else
+            string attachmentName = "InlineImage.jpg";
+            string contentType = MediaTypeNames.Image.Jpeg;
+            var content = new BinaryData("This is image file content.");
+            var contentId = "myInlineAttachmentContentId";
+#endif
+            var emailAttachment = new EmailAttachment(attachmentName, contentType, content);
+            emailAttachment.ContentId = contentId;
```

Comment:
was it deliberate to set the ContentId separately rather than use the constructor that takes in ContentId? I saw this pattern in all of the samples.

Reply:
Yes, since it's not a mandatory field.

Reply:
sounds good.